### PR TITLE
Export CSV

### PR DESCRIPTION
**Pull request by APalumbo97**

### CODE REVIEW

**Comment by APalumbo97** [comment]
Do not merge yet, this needs to be unit tested.

**Comment by APalumbo97** [comment]
Also make sure to squash and merge

File: db/schema/populate_trees.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	VALUES (2, 2, 1167);
+	
+INSERT INTO asset_property (asset_id, property_id, value)
+	VALUES (3, 2, 1277);
+	
+INSERT INTO asset_property (asset_id, property_id, value)
+	VALUES (4, 2, 1387);
+	
+INSERT INTO asset_property (asset_id, property_id, value)
+	VALUES (5, 2, 1497);
```
Comment (tilley14) [inline db/schema/populate_trees.sql]: newline

File: db/api/v1/db/assetDefinitions.db.js [hunk 2]
```diff
@@ -13,6 +16,27 @@ const findAssetTypes = async () => {
     return global.dbPool.query(query);
 };
 
+/**
+ * Query to find the property types for a given assetTypeID.
+ * @param {*} assetTypeID - the asset type ID
```
Comment (jedjas12) [inline db/api/v1/db/assetDefinitions.db.js]: Replace the star with the correct var type

File: salesforce/my-conservation-life/force-app/main/default/lwc/exportCSV/exportCSV.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        var i, j;
+        var csv_data = '';
+        var rows = [];
+        var props = {};
+        var keys;
+        var hiddenElement;
+
+        assetDefinitions.fetchAssetPropTypes(this.valueID)
+            .then(properties => {
+                rows = ['asset_type_id', this.value.split(':')[0].trimRight()];
```
Comment (tilley14) [inline salesforce/my-conservation-life/force-app/main/default/lwc/exportCSV/exportCSV.js]: comment on why your doing this

File: db/api/v1/routes/index.js [hunk 4]
```diff
@@ -22,6 +22,10 @@ router.get(
 
 router.post('/assets', assets.create);
 
+router.get('/asset_types', assetDefinitions.getAssetTypes);
```
Comment (tilley14) [inline db/api/v1/routes/index.js]: Can we make these camel case like the other endpoints in the db?

**Pull request merged**